src/middleware/devtools.ts

```diff
@@ -147,13 +148,41 @@ type DevtoolsImpl = <T>(
 
 export type NamedSet<T> = WithDevtools<StoreApi<T>>['setState']
 
+type Connection = ReturnType<
+  NonNullable<Window['__REDUX_DEVTOOLS_EXTENSION__']>['connect']
+>
+type ConnectionName = string | undefined
+type StoreName = string
+type StoreInformation = {
+  api: StoreApi<unknown>
+  initialState: unknown
```

Comment:
I'm not confident if we need this `initialState`. Can it be done with "current state" i.e. `api.getState()`?

Reply:
Think I managed to hammer this away in the aforementioned PR to the fork.